nym-sphinx-wasm/www/index.js

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 Nym Technologies SA
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as wasm from "nym-sphinx-wasm";
+
+console.log("Retrieving topology...");
```

Comment:
Just a minor thing, but shouldn't that be called in `getTopology()` function rather than on module import?

Reply:
Which, the console.log?

Reply:
"console.log("Retrieving topology...");"

Reply:
All this js is going to get banged around real soon, no fear there